2022-10-11 11:23:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Numeric value out of range: "2147483649" in column "AGE"; SQL statement:
INSERT INTO Employee(name, city, department, age)
                                                        VALUES ( ? ,  ? ,  ? ,  ? ) [22004-206]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:525)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.value.Value.convertToInt(Value.java:2690)
	at org.h2.value.Value.convertToInt(Value.java:1590)
	at org.h2.value.Value.convertTo(Value.java:1135)
	at org.h2.value.Value.convertForAssignTo(Value.java:1092)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:369)
	at org.h2.table.Table.convertInsertRow(Table.java:917)
	at org.h2.command.dml.Insert.insertRows(Insert.java:167)
	at org.h2.command.dml.Insert.update(Insert.java:135)
	at org.h2.command.CommandContainer.executeUpdateWithGeneratedKeys(CommandContainer.java:246)
	at org.h2.command.CommandContainer.update(CommandContainer.java:167)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:209)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:169)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at io.ballerina.stdlib.sql.nativeimpl.ExecuteProcessor.nativeExecuteExecutable(ExecuteProcessor.java:119)
	at io.ballerina.stdlib.sql.nativeimpl.ExecuteProcessor.lambda$nativeExecute$0(ExecuteProcessor.java:81)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
